guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,27 +37,36 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
+.Procedure
+. Install Discovery on {ProjectServer}:
++
+ifdef::foreman-el,katello[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=true \
+--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
```

Comment:
@evgeni I'd like to get your thoughts on this. The specific URL is copied from the existing documentation. If we compare it to the default `latest` (https://github.com/theforeman/puppet-foreman_proxy/blob/6bd3c34a0771bd87b6640360ad8f216b0b285d77/manifests/plugin/discovery/params.pp#L6) I wonder what's better. Using `latest` could be problematic for older releases, but on the other hand, we only support the last 2 releases and I don't really expect compatibility issues. Additionally, I don't expect users to change this on upgrade so should we recommend to omit this option and rely on the default?

Reply:
@mjivraja then please drop this line and `--foreman-proxy-plugin-discovery-install-images=true` (since that's a default too). Then remove that too in `guides/common/modules/proc_installing-the-discovery-service.adoc` to keep it consistent.

Reply:
install images is false by default:
https://github.com/theforeman/puppet-foreman_proxy/blob/6bd3c34a0771bd87b6640360ad8f216b0b285d77/manifests/plugin/discovery/params.pp#L4

Reply:
Oh right, thanks for checking.